storage/Devices/MdImpl.cc

```diff
@@ -291,25 +313,136 @@ namespace storage
     }
 
 
+    void
+    Md::Impl::calculate_region_and_topology()
+    {
+	vector<BlkDevice*> devices = get_devices();
+
+	long real_chunk_size = chunk_size;
+
+	if (real_chunk_size == 0)
+	    real_chunk_size = get_default_chunk_size();
+
+	// mdadm uses a chunk size of 64 KiB just in case the RAID1 is ever reshaped to RAID5.
+	if (md_level == RAID1)
+	    real_chunk_size = 64 * KiB;
+
+	int number = 0;
+	unsigned long long sum = 0;
```

Comment:
Yes, but a simple typedef does not allow for nice features, e.g. function overloading (e.g. ostream<<).

Addind a class for that has been on the TODO list for quite some time: https://trello.com/c/ub28GvPD/56-dedicated-class-disksize